docs/project-administration/releasing/mathics-core.rst

```diff
@@ -0,0 +1,142 @@
+==============================
+Making a Mathics3 Core Release
+==============================
+
+.. code::
+
+    $ PACKAGE="Mathics3"
+    $ package="mathics-core"
+    # Edit mathics/version.py
+    $ source mathics/version.py # to set in POSIX shell
+    $ echo $__version__
+
+Workflows update?
+=================
+
+Check ``.github/workflows/*.yml`` to make see if we are using
+github versions (of scanner) for testing. If so adjust.
+
+
+Create a new branch
+===================
+
+.. code::
+
+    $ git checkout -b release-$__version__
+    $ git commit -m"Get ready for release $__version__" .
+
+Update Changes
+==============
+
+.. code::
+
+    $ make ChangeLog
+
+Update ``CHANGES.rst`` from ``ChangeLog``
+
+::
+
+    $ make check
+    $ git commit --amend .
+    $ git push -u origin HEAD # get CI testing going early
+
+Make Python PDF
+======================
+
+.. code::
+
+   $ pyenv local 3.11 # or whatever latest Python you want ot build doc is
+   $ make clean # make sure mathics-title.pdf has not been removed
+   $ make doc
+
+
+Look at resulting PDF and colophon
+
+Check Python versions
+======================
+
+.. code::
+
+   $ ./admin-tools/check-versions.sh
+
+Merge release
+=============
+
+Go to github and merge release into master.
+
+Then...
+.. code::
+::
+
+    $ git checkout master
+    $ git pull
+
+
+Check package from github
+=========================
+
+TODO: turn this into a script in ``admin-tools``
+
+.. code::
+
+    $ [[ ! -d /tmp/gittest ]] && mkdir /tmp/gittest; pushd /tmp/gittest
+    $ pyenv local pyston-2.3.5 # Use a version that is not the most recent
```

Comment:
Just a silly comment: what about if you use conda in your system?

Reply:
I don't know much about conda. How would this work?

Reply:
Sure. I will look at it over the weekend.

Reply:
> Sure. I will look at it over the weekend.

Feel free to just go and adjust the instructions if you find a better way or more natural way to do things.